graphql/pod_resolver.go

```diff
@@ -41,6 +43,28 @@ func (r *podResolver) Task(ctx context.Context, obj *model.APIPod) (*model.APITa
 	return apiTask, nil
 }
 
+func (r *podResolver) Events(ctx context.Context, obj *model.APIPod, limit *int, page *int) (*PodEvents, error) {
+	events, count, err := event.MostRecentPaginatedPodEvents(utility.FromStringPtr(obj.ID), *limit, *page)
```

Comment:
Can `limit` or `page` ever be nil, or will GraphQL always ensure they're a non-nil value?